src/common/tweaksManager.js

```diff
@@ -268,10 +268,11 @@ TweaksManager.NON_COMPATIBLE_TAGS = ["TR", "TH", "TD", "THEAD", "TBODY", "TFOOT"
             try {
                 r = !!window.parent.document.title.match(/roundcube/i) || !!window.parent.location.href.match(/_action=compose/i)
             } catch (e) {}
-            const n = isProseMirror(e),
-                o = isCKEditor(e),
-                i = isQuillEditor(e),
-                l = isLTEditor(e);
+            // Direct checks instead of using site-specific functions (issue #192)
+            const n = e.classList.contains("ProseMirror"),
+                o = e.classList.contains("cke_editable"),
+                i = e.classList.contains("ql-editor"),
+                l = e.classList.contains("lt-textarea__textarea");
```

Comment:
Can you remove the variables, `n`, `o`, `i`, `l` and code using them? We don't know what this code does, see if you can just remove it.